code/zFulpstationCode/fulp_overwrite_vars.dm

```diff
@@ -446,3 +446,34 @@
 //**** Sleepy Pen for ops ENDS - Glubtok, Jul 2020
 //***********************************************************
 
+
+
+//***********************************************************
+//****UNSTAGRAV HOLODRAGON  PR *****************************
+//***********************************************************
+
+/obj/item/clothing/under/rank/engineering/atmospheric_technician
+	desc = "It's a jumpsuit worn by atmospheric technicians. It is protected from fire, not sure about you tho."
+	name = "atmospheric technician's jumpsuit"
+	icon_state = "atmos"
+	inhand_icon_state = "atmos_suit"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 10, "fire" = 0, "acid" = 20)
+	resistance_flags = null // the tech suit will never burn now which is essential to make this work.
```

Comment:
you, uh, 100% sure about that?

Reply:
ya because I lit myself on fire and it didn't do anything.  then I welderbombed myself after and my pants flew off.